Packs/CommonScripts/Scripts/ExtractIndicatorsFromTextFile/ExtractIndicatorsFromTextFile.py

```diff
@@ -3,6 +3,22 @@
 from CommonServerUserPython import *
 
 
+def read_file_with_encoding_detection(filePath, maxFileSize):
+    encoding_types = ['utf-8', 'ISO-8859-9', None]  # use None to simulate open file with no encoding sepcified
```

Comment:
when do we want no encoding specified? What default encoding does open use?

Reply:
I left this more as a safety measure to mimic previous behavior.
Seems encoding is based on the platform you use, this is from the documentation-
```
* On Android and VxWorks, return "utf-8".

* On Unix, return the encoding of the current [LC_CTYPE](https://docs.python.org/3/library/locale.html#locale.LC_CTYPE) locale. Return "utf-8" if nl_langinfo(CODESET) returns an empty string: for example, if the current LC_CTYPE locale is not supported.

* On Windows, return the ANSI code page.
```
Should I leave it as is in case the system is run on a Windows server?

Reply:
we run inside a docker container so that shouldnt change. native image and python3 have the same lctype. we should take out none. There isnt a real reason why we would want this to be enviornment dependant

Reply:
Cool will remove